www/src/templates/template-blog-post.js

```diff
@@ -1,170 +1,282 @@
 import React from "react"
 import Helmet from "react-helmet"
+import Link from "gatsby-link"
+import ArrowForwardIcon from "react-icons/lib/md/arrow-forward"
+import ArrowBackIcon from "react-icons/lib/md/arrow-back"
 
 import presets from "../utils/presets"
-import typography, { rhythm, scale } from "../utils/typography"
+import typography, { rhythm, scale, options } from "../utils/typography"
 import Container from "../components/container"
 
 const BlogPostTemplate = React.createClass({
   render() {
     const post = this.props.data.markdownRemark
+    const prev = this.props.pathContext.prev
+    const next = this.props.pathContext.next
+    const prevNextLinkStyles = {
+      "&&": {
+        boxShadow: `none`,
+        borderBottom: 0,
+        fontFamily: options.headerFontFamily.join(`,`),
+        fontWeight: `bold`,
+        color: presets.brand,
+      },
+    }
+    const prevNextLabelStyles = {
+      marginTop: 0,
+      marginBottom: 0,
+      color: `rgba(0,0,0,.30)`,
+      color: `rgba(38, 32, 44,.44)`,
+      fontWeight: `normal`,
+      ...scale(0),
+      lineHeight: 1,
+    }
     const BioLine = ({ children }) =>
       <p
         css={{
           ...scale(-2 / 5),
           fontFamily: typography.options.headerFontFamily.join(`,`),
-          lineHeight: 1.3,
+          lineHeight: 1.2,
           margin: 0,
           color: `rgba(0,0,0,.44)`,
+          color: `rgba(38, 32, 44,.62)`,
           [presets.Mobile]: {
             ...scale(-1 / 5),
-            lineHeight: 1.3,
+            lineHeight: 1.2,
           },
         }}
       >
         {children}
       </p>
+    const headLinks = [
+      {
+        rel: `author`,
+        href: `https://gatsbyjs.org${post.frontmatter.author.slug}`,
+      },
+    ]
+    if (post.frontmatter.canonicalLink) {
+      headLinks.push({
+        rel: `canonical`,
+        href: post.frontmatter.canonicalLink,
+      })
+    }
+    console.log(headLinks)
     return (
-      <Container>
-        {/* Add long list of social meta tags */}
-        <Helmet
-          title={post.frontmatter.title}
-          link={[
-            {
-              rel: `author`,
-              href: `https://gatsbyjs.org${post.frontmatter.author.slug}`,
```

Comment:
Thinking about how the author page would look/how author bios would look on blog post pages once we have a dedicated author page in place…

Should we just crop the author bio on the blog post page or distinguish between "short bio" and "author page bio" and offer fields for both?

I saw that there's already field for Twitter in authors.yaml already so we can at least show a Twitter icon on the author page. Maybe we should add some fields for the blog info (URL, title)? What about an (optional) link to a Github profile, too?